recipes/perl-capture-tiny/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "0.48" %}
+
+package:
+  name: perl-capture-tiny
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/Capture-Tiny-{{ version }}.tar.gz
+  sha256: 6c23113e87bad393308c90a207013e505f659274736638d8c79bac9c67cc3e19
+
+build:
+  noarch: generic
```

Comment:
```suggestion
```
We can't have `noarch: generic` Perl packages, yet (buthopefully soon-ish).
Building a `noarch: generic` package on Linux would make it installable but not runnable on other platforms, e.g., Windows. So for now, we have to have them in the platform-specific subdirs, not `noarch`.

Reply:
Thanks for reviewing. Deleted this line, squashed, and force-pushed.